civis/cli/_cli_commands.py

```diff
@@ -240,7 +240,7 @@ def notebooks_download_cmd(notebook_id, path):
     """Download a notebook to a specified local path."""
     client = civis.APIClient()
     info = client.notebooks.get(notebook_id)
-    response = requests.get(info['notebook_url'], stream=True)
+    response = requests.get(info['notebook_url'], stream=True, timeout=60)
```

Comment:
The `bandit` check got upset if we didn't have timeout set for `requests` calls. See similar changes like [this](https://github.com/civisanalytics/civis-python/pull/467/files#diff-d5eeabf5f99b39028dc56ea567236251f0c3b8c204d4f27b96ff6df449b8cd04R243) done to the alternate `v1` branch.